### ISSUE TITLE
W3C error:  Element input is missing required attribute aria-expanded.

### ISSUE THREAD

**Issue by ghost** [magento/magento2#2681]
W3C error on fresh Magento 2 installation :

<input id="search"↩                           data-mage-init='{"quickSearch":{↩                                "formSelector":"#search_mini_form",↩                                "url":"http://yourwebsite.com/search/ajax/suggest/",↩                                "destinationSelector":"#search_autocomplete"}↩                           }'↩                           type="text"↩                           name="q"↩                           value=""↩                           placeholder="Search entire store here..."↩                           class="input-text"↩                           maxlength="128"↩

Attributes for element input:
Global attributes  
accept  when type is file
alt when type is image
autocomplete    when type is text, search, url, tel, e-mail, password, datetime, date, month, week, time, datetime-local, number, range, or color
autofocus  
checked when type is checkbox or radio
dirname when type is text or search
disabled  
form  
formaction  when type is submit or image
formenctype when type is submit or image
formmethod  when type is submit or image
formnovalidate  when type is submit or image
formtarget  when type is submit or image
height  when type is image
list    when type is text, search, url, tel, e-mail, datetime, date, month, week, time, datetime-local, number, range, or color
max when type is datetime, date, month, week, time, datetime-local, number, or range
maxlength   when type is text, search, url, tel, e-mail, or password
min when type is datetime, date, month, week, time, datetime-local, number, or range
multiple    when type is email or file
name  
pattern when type is text, search, url, tel, e-mail, or password
placeholder when type is text, search, url, tel, e-mail, password, or number
readonly    when type is text, search, url, tel, e-mail, password, datetime, date, month, week, time, datetime-local, or number
required    when type is text, search, url, tel, e-mail, password, datetime, date, month, week, time, datetime-local, number, checkbox, radio, or file
size    when type is text, search, url, tel, e-mail, or password
src when type is image
step    when type is datetime, date, month, week, time, datetime-local, number, or range
type  
value   when type is not file or image
width   when type is image

W3C not compliant.

**Comment by odubovyk** [comment]
Thank you for reporting this issue! Internal ticket MAGETWO-50628

**Comment by vil11** [comment]
FYI: https://github.com/magento/magento2/wiki/Issue-reporting-guidelines

Dmytro Vilchynskyi, M2 QA.

**Comment by MomotenkoNatalia** [comment]
Internal ticket MAGETWO-50628 is closed and delivered to mainline. Please let us know if you still have an issue.

**Comment by odubovyk** [comment]
Ghost, please note that the attribute is added via JS and original source never changes. Only the DOM changes by JS.
You can see an HTML visualization of DOM changes using your browser's developer tools.

**Comment by ooples** [comment]
I'm getting this error and I'm using magento 2.1.6
